src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -12,8 +12,23 @@ export class SavedObjectsClient {
     this._callAdminCluster = callAdminCluster;
   }
 
-  async create(type, body = {}) {
-    const response = await this._withKibanaIndex('index', { type, body });
+  /**
+   * Persists an object
+   *
+   * @param {string} type
+   * @param {object} body - { attributes: {}, id: myId }
+   * @param {object} options
+   * @param {boolean} options.overwrite - defaults to false
+   * @returns {promise}
+  */
+  async create(type, body = {}, options = {}) {
```

Comment:
The `body` argument here feels out of place. Before we had the concept of `attributes`, `body` was used to describe the the same thing, the key/value pairs that define the contents of a saved object. Now it seems that `body` represents the API request body (more or less), which isn't really relevant to the `SavedObjectsClient`. I think we should update this to use the same call signature as `savedObjectsClient.update(type, id, attributes, options)`;